prover/crypto/ringsis/ringsis_test.go

```diff
@@ -40,28 +40,13 @@ var testCasesKey = []struct {
 		Size:   576,
 		Params: StdParams,
 	},
-	// TODO @gbotrel confirm with @AlexandreBelling we don't need to test these cases
-	// {
-	// 	Size: 43,
-	// 	Params: Params{
-	// 		LogTwoBound:  1,
-	// 		LogTwoDegree: 1,
-	// 	},
-	// },
-	// {
-	// 	Size: 23,
-	// 	Params: Params{
-	// 		LogTwoBound:  1,
-	// 		LogTwoDegree: 1,
-	// 	},
-	// },
-	// {
-	// 	Size: 256,
-	// 	Params: Params{
-	// 		LogTwoBound:  1,
-	// 		LogTwoDegree: 1,
-	// 	},
-	// },
+	{
```

Comment:
We can't just keep the "production" case. It has to work for all parameters even if it means that we use a fallback, inefficient implementation for those.

Reply:
so, the only decision we made was limb size (bound) has to be a multiple of 8 (byte size). Rest works fine, this PR updates gnark-crypto because previously we incorrectly used the fast path in some cases we shouldn't have

Reply:
Since the SIS initialization was incorrectly using the fast path in some cases, does it need to extend test cases to cover edge cases explicitly?
Specifically, verifying that the fallback implementation works correctly in non-standard parameter cases could prevent future regressions.